src/test/java/edu/ucla/library/iiif/auth/handlers/AccessTokenHandlerIT.java

```diff
@@ -51,26 +53,42 @@ public final class AccessTokenHandlerIT extends AbstractAccessTokenHandlerIT {
     /**
      * Tests that a browser client can use a valid access cookie to obtain an access token.
      *
+     * @param aReverseProxyDeployment Whether or not to simulate app deployment behind a reverse proxy
      * @param aVertx A Vert.x instance
      * @param aContext A test context
      */
-    @Test
-    public void testGetTokenBrowser(final Vertx aVertx, final VertxTestContext aContext) {
+    @ParameterizedTest
+    @ValueSource(booleans = { true, false })
+    public void testGetTokenBrowser(final boolean aReverseProxyDeployment, final Vertx aVertx,
+            final VertxTestContext aContext) {
         final String getCookieRequestURI =
                 StringUtils.format(GET_COOKIE_PATH, URLEncoder.encode(TEST_ORIGIN, StandardCharsets.UTF_8));
         final HttpRequest<?> getCookie = myWebClient.get(myPort, TestConstants.INADDR_ANY, getCookieRequestURI);
 
+        if (aReverseProxyDeployment) {
+            getCookie.putHeader(X_FORWARDED_FOR, FORWARDED_IP_ADDRESSES);
+        }
+
         getCookie.send().compose(result -> {
             final String cookieHeader = result.cookies().get(0);
             final String cookieValue = cookieHeader.split(EQUALS)[1];
-            final String clientIpAddress =
-                    Jsoup.parse(result.bodyAsString()).getElementById(myClientIpAddressID).text();
+            final String clientIpAddress;
+
+            if (aReverseProxyDeployment) {
+                clientIpAddress = FORWARDED_CLIENT_IP;
+            } else {
+                clientIpAddress = Jsoup.parse(result.bodyAsString()).getElementById(myClientIpAddressID).text();
+            }
```

Comment:
Is there an ENV to set aReverseProxyDeployment?

Reply:
I have it turned on by default: https://github.com/UCLALibrary/hauth/pull/40/files#diff-337ee99aa6dbf4b18b70e2c73ec27930d8344c2e7e9d696d20d26e160d46caafR166-R168

Reply:
We could make it toggle-able with an env var if you want, but essentially all this PR does is add the `X-Forwarded-For` header to the list of places where the client IP address may be found. If that header isn't present, then the app behaves as if this change was never made.

Reply:
The current implemented behavior is fine. Let's proceed. 👍